src/store.ts

```diff
@@ -144,17 +144,20 @@ export class Store<T> {
     }
     PLATFORM.performance.mark("dispatch-start");
 
-    const action = this.actions.get(reducer);
+    const action = {
+      type: this.actions.get(reducer)!.type,

```

Comment:
```suggestion
      ...this.actions.get(reducer)!,
```

this way, if for whatever reason more props get added we'll capture them all

Reply:
How should we represent that change in the `Action<T = any>` interface? Typescript recommends a nested property to represent index signatures, but you may have a better way. Also, I realized i forgot to add the `params: (...args: any[]);` property, which i can add in another commit on this PR.

Reply:
@zewa666 just want to make sure you saw my last question and it was not missed since i had two reply's.

Reply:
Im not sure about which part we're talking here. Didnt notice anything missing or errors from TS. but just do the change I'm pretty sure youll do great. We can review afterwards.